src/align_func_proto.cpp

```diff
@@ -12,6 +12,11 @@
 #include "align_stack.h"
 #include "align_tools.h"
 #include "log_rules.h"
+#include "LIMITATIONS.h"
+
+#ifdef WIN32
```

Comment:
Semi-pedantic: `<algorithm>` if the correct header for `std::max`. It probably comes in as a transitive include on other platforms, but if it's used, it's "correct" to just `#include <algorithm>`. (IOW, I would drop the `#ifdef WIN32`.)